StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1013ClosingCurlyBracketsMustBeSpacedCorrectly.cs

```diff
@@ -77,26 +78,12 @@ private void HandleCloseBraceToken(SyntaxTreeAnalysisContext context, SyntaxToke
                 return;
             }
 
-            bool precededBySpace;
-            bool firstInLine;
-
-            bool followedBySpace;
-            bool lastInLine;
+            bool firstInLine = token.IsFirstInLine();
+            bool precededBySpace = firstInLine ? true : token.IsPrecededBySpace();
```

Comment:
You could also write this as `precededBySpace = firstInLine || token.IsPrecededBySpace();`